models/routes_test.go

```diff
@@ -4,37 +4,63 @@ import (
 	"encoding/json"
 
 	"code.cloudfoundry.org/bbs/models"
-	"github.com/gogo/protobuf/proto"
+	"google.golang.org/protobuf/proto"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 )
 
 var _ = Describe("Routes", func() {
 	var update models.DesiredLRPUpdate
-	var aJson models.DesiredLRPUpdate
+	var aJson models.ProtoDesiredLRPUpdate
 	var aProto models.DesiredLRPUpdate
+	var resultProto models.ProtoDesiredLRPUpdate
 
 	itSerializes := func(routes *models.Routes) {
 		BeforeEach(func() {
-			update = models.DesiredLRPUpdate{
-				Routes: routes,
+
+			// don't really care for branching logic in tests, but don't want to change the test right now
+			if routes == nil {
+				update = models.DesiredLRPUpdate{
+					Routes: nil,
+				}
+			} else {
+				update = models.DesiredLRPUpdate{
+					Routes: routes,
+				}
```

Comment:
This logic seems redundant. Can't you just have a single update = <thing> block here? if routes is nil, it wll assign nil to Routes anyway.

Reply:
This is an artifact from when `models/routes.go` was in a state of flux.  The single assignment should be fine.

Reply:
Cleaned up in [f7314a9](https://github.com/cloudfoundry/bbs/pull/109/commits/f7314a95de15266d952902af8767dd6b5a741df3)